src/scripts/quests/QuestLineHelper.ts

```diff
@@ -2472,6 +2472,43 @@ class QuestLineHelper {
         App.game.quests.questLines().push(unrivaledPowerQuestLine);
     }
 
+    public static createLightsCameraPikaQuestLine() {
+        const lightsCameraPikaQuestLine = new QuestLine('Lights! Camera! Pika!', 'Frank is making a new movie, and needs your help with finding his stars.');
+
+        const trainPikachuQuest = new TrainSpecificPokemonQuest('Pikachu', 50000);
+        lightsCameraPikaQuestLine.addQuest(trainPikachuQuest);
+
+        const talkToFrankQuest1 = new TalkToNPCQuest(DirectorFrank2, 'Return to Frank');
+        lightsCameraPikaQuestLine.addQuest(talkToFrankQuest1);
+
+        const trainPikachuKalosQuest = new TrainSpecificPokemonQuest('Pikachu (Kalos Cap)', 50000);
+        lightsCameraPikaQuestLine.addQuest(trainPikachuKalosQuest);
+
+        const talkToFrankQuest2 = new TalkToNPCQuest(DirectorFrank3, 'Return to Frank');
+        lightsCameraPikaQuestLine.addQuest(talkToFrankQuest2);
+
+        const trainHawluchaQuest = new TrainSpecificPokemonQuest('Hawlucha', 50000).withCustomReward(() => {
+            App.game.party.gainPokemonByName('Hawlucha (Stuntman)', PokemonFactory.generateShiny(GameConstants.SHINY_CHANCE_REWARD));
+        });
+        lightsCameraPikaQuestLine.addQuest(trainHawluchaQuest);
+
+        const talkToFrankQuest3 = new TalkToNPCQuest(DirectorFrank4, 'Return to Frank');
+        lightsCameraPikaQuestLine.addQuest(talkToFrankQuest3);
+
+        const trainPikachusQuest = new CustomQuest(15, undefined, 'Train 15 Pikachus to 25.000 power',
```

Comment:
Should `power` be attack? Also I think these quests aren't working. They all increased every time my Pikachu leveled up.